src/test/java/com/hotjava/app/hotjava/HotJavaApplicationTests.java

```diff
@@ -1,234 +0,0 @@
-	void contextLoads() {
-	}
-
-	/**
-	 * Validate that the DTO properties for Vehicle can be set and retrieved.
-	 */
-	@Test
-	void verifyVehicleProperties(){
-		int vehicleSubmissionID = 8675309;
-		 String vehicleOwnerName = "test name";
-		 String vehicleDescription ="test desc";
-		 String vehicleYear = "1999";
-	     String vehicleMake = "test make";
-		 String vehicleModel = "test model";
-		 String vehicleScore = "test score";
-		 Photo vehiclePhoto = new Photo();
-		 vehiclePhoto.setPhotoId(111);
-
-		Vehicle testVehicle = new Vehicle();
-		testVehicle.setVehicleSubmissionID(vehicleSubmissionID);
-		testVehicle.setVehicleOwnerName(vehicleOwnerName);
-		testVehicle.setVehicleDescription(vehicleDescription);
-		testVehicle.setVehicleYear(vehicleYear);
-		testVehicle.setVehicleMake(vehicleMake);
-		testVehicle.setVehicleModel(vehicleModel);
-		testVehicle.setVehicleScore(vehicleScore);
-		testVehicle.setPhoto(vehiclePhoto);
-		vehiclePhoto.setPhotoId(111);
-		vehiclePhoto.setFileName("civic-type-r.jpg");
-		vehiclePhoto.setPath("src/main/resources/img/civic-type-r.jpg");
-
-		assertEquals(vehicleSubmissionID,testVehicle.getVehicleSubmissionID());
-		assertEquals(vehicleOwnerName,testVehicle.getVehicleOwnerName());
-		assertEquals(vehicleDescription,testVehicle.getVehicleDescription());
-		assertEquals(vehicleYear,testVehicle.getVehicleYear());
-		assertEquals(vehicleMake,testVehicle.getVehicleMake());
-		assertEquals(vehicleModel,testVehicle.getVehicleModel());
-		assertEquals(vehicleScore,testVehicle.getVehicleScore());
-		assertEquals(vehiclePhoto, testVehicle.getPhoto());
-
-	}
-
-	/**
-	 * Validate that the DAOStub and service can add vehicles.
-	 */
-	@Test
-	void verifyServiceCanAddVehicles() throws Exception{
-		int vehicleSubmissionID = 8675309;
-		String vehicleOwnerName = "test name";
-		String vehicleDescription ="test desc";
-		String vehicleYear = "1999";
-		String vehicleMake = "test make";
-		String vehicleModel = "test model";
-		String vehicleScore = "test score";
-		Photo vehiclePhoto = new Photo();
-		vehiclePhoto.setPhotoId(111);
-		vehiclePhoto.setFileName("civic-type-r.jpg");
-		vehiclePhoto.setPath("src/main/resources/img/civic-type-r.jpg");
-
-		Vehicle testVehicle = new Vehicle();
-		testVehicle.setVehicleSubmissionID(vehicleSubmissionID);
-		testVehicle.setVehicleOwnerName(vehicleOwnerName);
-		testVehicle.setVehicleDescription(vehicleDescription);
-		testVehicle.setVehicleYear(vehicleYear);
-		testVehicle.setVehicleMake(vehicleMake);
-		testVehicle.setVehicleModel(vehicleModel);
-		testVehicle.setVehicleScore(vehicleScore);
-		testVehicle.setPhoto(vehiclePhoto);
-
-		vehicleServiceNoMock.save(testVehicle);
-
-		List<Vehicle> vehicleList = vehicleServiceNoMock.fetchAll();
-
-		Boolean vehiclePresent = false;
-		for (Vehicle v: vehicleList) {
-			if(v.getVehicleSubmissionID() == vehicleSubmissionID){
-				if(v == testVehicle){
-					vehiclePresent = true;
-				}
-			}
-		}
-        assertTrue(vehiclePresent);
-	}
```

Comment:
Since the DAO was mocked this the service will not reach the stub.
Additionally, this is an integration test and should be separated from the unit tests.